tools/mmp/driver.cs

```diff
@@ -366,6 +374,22 @@ static void Main2 (string [] args)
 						aotOptions = new AOTOptions (v);
 					}
 				},
+				{ "native-strip=", "Strip native libraries copied into bundle of their unnecessary architectures. Defaults to true in Release and False in debug. (default, strip, skip)", v => {
+					switch (v) {
+					case "default":
+						NativeStrip = NativeStripMode.Default;
+						break;
+					case "strip":
+						NativeStrip = NativeStripMode.Strip;
+						break;
+					case "skip":
+						NativeStrip = NativeStripMode.Skip;
+						break;
+					default:
+						ErrorHelper.Error (26, $"Could not parse the command line argument '-native-strip:{v}'");
+						break;
+					}
```

Comment:
I wonder if that should be added in the new optimizations ? c.c. @rolfbjarne 
It behaves just like them, i.e. a sane *default* (computed based on the project settings) and overrides for *enabling* and *disabling* it.
In any case it needs to be documented outside the errors code it introduce.